docs/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Added
 
 - Add support for Robot Framework v4 XML files. Closes [#2136](https://github.com/ICTU/quality-time/issues/2136).
+- Add 'Missing metrics' metric. Closes [#1477](https://github.com/ICTU/quality-time/issues/1477).
```

Comment:
This is not the correct location v3.22.0 was released yesterday, so you should add a new [Unreleased] header above [3.22.0].